spanner_orm/admin/migration.skel

```diff
@@ -16,3 +16,12 @@ def upgrade() -> spanner_orm.NoUpdate:
 # Returns a SchemaUpdate object that tells how to roll back the changes
 def downgrade() -> spanner_orm.NoUpdate:
   return spanner_orm.NoUpdate()
+
+# Note: upgrade/downgrade can also return a list of SchemaUpdates
+#       which will each be executed in sequential order
```

Comment:
I'd add a note here that downgrade should reverse the list of operations.

Reply:
good call 👌